set_up_cbmc_runner/action.yml

```diff
@@ -31,49 +31,97 @@ runs:
       uses: actions/checkout@v3
       with:
         submodules: 'recursive'
-    - name: Install latest CBMC
+
+    - env:
+        bashPass: \033[32;1mPASSED -
+        bashInfo: \033[33;1mINFO -
+        bashFail: \033[31;1mFAILED -
+        bashEnd:  \033[0m
+        stepName: Install latest CBMC
+      name: ${{ env.stepName }}
       if: ${{ inputs.cbmc_version == 'latest' }}
       shell: bash
       run: |
-        echo "Using latest CBMC"
+        # ${{ env.stepName }}
+        echo -e "::group::${{ env.bashInfo }} ${{ env.stepName }} ${{ env.bashEnd }}"
         # Search within 5 most recent releases for latest available package
```

Comment:
Is this comment still accurate?

Reply:
Yeah, the comment went from `Using latest CBMC` to `Install latest CBMC`